src/consensus/ledgerenclave.h

```diff
@@ -83,6 +83,17 @@ namespace consensus
       serialized::skip(data, size, entry_len);
     }
 
+    std::pair<std::vector<uint8_t>, bool> get_entry(
```

Comment:
needed as I want the entry to desrialise it but I don't want to write to the ledger yet since I am re-executing

Reply:
This could be merged with `skip_entry()` above as the two functions basically do the same thing. Also, I don't think it needs the return a pair as the second element is always `true`.

Reply:
not sure we want `skip_entry` to always return the entry that it skipped though, there are places where we just want to skip the entry, not copy the entry into a vector and return it